### PR TITLE
 🎉 1.0.0

### PR DESCRIPTION
**Pull request by ZihaoZhou**
Finishing up v1.0.0.

### CODE REVIEW

**Comment by VSumanth99** [comment]
Some suggestions for the README:
1. Maybe add a line to show how to modify the `PYTHONPATH`.
2. Add instructions for using the evaluation script to get the aggregate results.

Verified that the code runs with `gemini-1.5-flash`.

File: .gitignore
Comment (huiwenn) [inline .gitignore:1]: let's add .vscode?

**Review by huiwenn — APPROVED** [review]
Overall LGTM.

**Review by VSumanth99 — APPROVED** [review]
looks great!

**Pull request merged**